packages/zowe-explorer/src/trees/ZoweTreeProvider.ts

```diff
@@ -266,6 +266,7 @@ export class ZoweTreeProvider<T extends IZoweTreeNode> {
                 Profiles.getInstance().validProfile = Validation.ValidationType.UNVERIFIED;
             }
         }
+        await ZoweTreeProvider.checkJwtTokenForProfile(node.getProfileName());
```

Comment:
Curious if we'd want to move this inside `Profiles.getInstance().checkCurrentProfile` so that we can set the validation status to `inactive` when token is expired?

After further thought, maybe not and this is a dumb question 😋 I believe validation status is only used to determine if the host/port are accessible, so we don't need to update it for invalid or expired credentials.

Reply:
While I like this idea, I'm on the fence for changing this if what you mentioned about validation is true:

>  I believe validation status is only used to determine if the host/port are accessible

Maybe we can track this as a separate enhancement to get the team's consensus on updating the validating profiles logic? I don't believe we do this for basic credentials, as validation currently passes even with invalid credentials - the invalid credentials error does not appear until after the search is submitted. But if we want to do this for tokens, we should probably do something similar for basic creds 😋